rdmo/projects/forms.py

```diff
@@ -266,10 +266,15 @@ def __init__(self, *args, **kwargs):
 
         # add fields for the integration options
         for field in self.provider.fields:
-            try:
-                initial = IntegrationOption.objects.get(integration=self.instance, key=field.get('key')).value
-            except IntegrationOption.DoesNotExist:
+            # new integration instance is going to be created
```

Comment:
Is this the best to make the test pass?
It has pretty high complexity.

Reply:
I think its ok.